bundler/spec/commands/newgem_spec.rb

```diff
@@ -161,6 +161,26 @@ def ignore_paths
     end
   end
 
+  shared_examples_for "--bundle flag" do
+    before do
+      bundle "gem #{gem_name} --bundle"
+    end
+    it "generates a gem skeleton with bundle install" do
+      gem_skeleton_assertions
+      expect(Kernel).to receive(:system).with("bundle", "install").once
+    end
+  end
+
+  shared_examples_for "--no-bundle flag" do
+    before do
+      bundle "gem #{gem_name} --no-bundle"
+    end
+    it "generates a gem skeleton without bundle install" do
+      gem_skeleton_assertions
+      expect(Kernel).not_to receive(:system).with("bundle", "install")
+    end
+  end
```

Comment:
You added shared examples, but they are not getting run, right?

Reply:
fixed